src/components/SectionCampaign/examples.md

```diff
@@ -0,0 +1,17 @@
+## Examples
+
+### Basic usage with Shopify section rendering
+
+```html
+<nosto-section-campaign placement="front-page" section="product-recommendations"></nosto-section-campaign>
+```
+
+This example demonstrates basic integration with Shopify's Section Rendering API to display Nosto campaign results. The component fetches recommendations from the "front-page" placement and renders them using the "product-recommendations" section, maintaining consistency with your Shopify theme's styling and layout.
```

Comment:
provide also an example of a section liquid template that can be used with this custom element

the liquid template should take the product handles from the search.terms liquid variable, loop through them and render product cards